Userland/Libraries/LibCore/System.cpp

```diff
@@ -979,6 +979,21 @@ ErrorOr<void> adjtime(const struct timeval* delta, struct timeval* old_delta)
 }
 #endif
 
+#ifdef AK_OS_SERENITY
+ErrorOr<void> join_jail(int jail_index)
```

Comment:
Typically we add these things to both LibC/serenity.cpp and LibCore/System.cpp. I'm not sure if we need to keep that pattern up now that all our own apps are using LibCore, but if someone ports a FreeBSD program that knows how to use jails it might be nice to have a C API.

Reply:
To be honest, I am not aiming for feature-parity or even compatibility with FreeBSD jails in the long run, so I just neglected to add functions nobody will use into LibC.